zeppelin-client-examples/src/main/java/org/apache/zeppelin/client/examples/ZeppelinClientExample.java

```diff
@@ -56,6 +56,8 @@ public static void main(String[] args) throws Exception {
               "%python\nimport time\ntime.sleep(5)\nprint('done')");
       zClient.submitParagraph(noteId, paragraphId2);
       zClient.waitUtilParagraphRunning(noteId, paragraphId2);
+      // It's also ok here to call zClient.cancelNote(noteId);
```

Comment:
It looks like test for NotebookRestApi#stopNoteJobs is missing, it is better to add test for it. Also need to revisit whether we need to add test for other rest api methods